docs/product-release-notes/src/main/asciidoc/rn-whats-new-con.adoc

```diff
@@ -145,10 +131,13 @@ Initial integration with NoSQL has been introduced. This is based on additional
 The Quartz scheduler service has been improved to fetch jobs only for deployments (KIE containers) that are currently available in the running system. 
 
 == {KIE_SERVER}
-The {KIE_SERVER} (also known as kie-server) has been extended to support core engine features and offers a remote API for these operations. In addition, the following architectural changes were introduced.
+The {KIE_SERVER} (also known as the KIE server) has been extended to support core engine features and offers a remote API for these operations. In addition, the following architectural changes were introduced.
 
-=== Separate {CENTRAL} from execution server
-{CENTRAL} now delegates all of its requests to the execution server. The main advantage is that {CENTRAL} can now be used to monitor any set of execution servers. By linking the execution server to {CENTRAL}, the process and task monitoring interfaces in {CENTRAL} can now connect to this execution server and show all relevant information. When multiple independent execution servers are used, you can either connect to a specific one or use the smart router to aggregate information across multiple servers. 
+=== Separate {CENTRAL} from {KIE_SERVER}
+{CENTRAL} now delegates all of its requests to the {KIE_SERVER}. The main advantage is that {CENTRAL} can now be used to monitor any set of {KIE_SERVERS}. By linking the {KIE_SERVER} to {CENTRAL}, the process and task monitoring interfaces in {CENTRAL} can now connect to this {KIE_SERVER}  and show all relevant information. When multiple independent {KIE_SERVERS} are used, you can either connect to a specific one or use the smart router to aggregate information across multiple servers. 
```

Comment:
By linking the {KIE_SERVER} to {CENTRAL}, certain components of the CENTRAL can connect to KIE_SERVER? Connect by linking?..

Reply:
Also, you actually can connect multiple KIE Servers to a Central even without Smart Router (took me some time to find that out), but not sure if this needs to be here